ci-operator/step-registry/mount-ns-hiding/mount-ns-hiding-commands.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function create_mount_namespace_hiding_mc(){
+  # enabled can be either "true" or "false"
+  local enabled="$1"
+
+  local action="enable"
+  if [ "$enabled" = "false" ]; then
+    action="disable"
+  fi
+
+  local machineconfig=""
+
+  for role in master worker
+  do
+    echo "Rendering machineconfig to $action mount namespace hiding for $role node(s)"
+
+    str=$(cat << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: $role
+  name: 99-custom-enable-kubens-$role
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - enabled: $enabled
+        name: kubens.service
+---
+EOF
+)
+    machineconfig+="${str}\n"
+  done
+
+  echo "Applying machineconfigs..."
+  echo -en "$machineconfig" | oc create -f -
+
+  echo "Waiting for machineconfigs to begin rolling out"
+  oc wait --for=condition=Updating mcp/$role --timeout=5m
+
+  echo "Waiting for machineconfigs to finish rolling out"
+  oc wait --for=condition=Updated mcp/$role --timeout=30m
```

Comment:
At this point role=worker, so this will wait for worker but not master; perhaps this wait should be in a loop to ensure we wait for both roles?

Reply:
Good catch! Putting the `oc wait ...` condition(s) in a loop reduces the solution to what I had previously and thus negating the advantage of applying the MCs concurrently. This works better: `oc wait --for=condition=Updating mcp --all --timeout=5m`

Reply:
Validation that my proposed change works as expected:

<img width="957" alt="Screen Shot 2022-08-29 at 12 17 44 PM" src="https://user-images.githubusercontent.com/3267755/187247375-514988ca-4759-42a5-99a2-a7b84eb416ec.png">